Jenkinsfile

```diff
@@ -57,17 +57,14 @@ pipeline {
             }
         }
 
-        /*
-        TODO uncomment and finish up the command here once we're ready to release snapshots
         stage('Publish snapshot') {
             when {
                 branch 'master'
             }
             steps {
-                sh './mvnw $TODO'
+                sh './mvnw clean deploy -Papache-release'
```

Comment:
This will run the whole build again. Do we want to maybe just run the Maven Deploy Plugin's `deploy:deploy` goal instead since the previous stage would have done the whole build except for deploying?
https://maven.apache.org/plugins/maven-deploy-plugin/

Reply:
#21 :)